examples/wave-eager.py

```diff
@@ -107,21 +107,49 @@ def rhs(t, w):
     t = 0
     t_final = 3
     istep = 0
-    while t < t_final:
-        fields = rk4_step(fields, t, dt, rhs)
 
-        if istep % 10 == 0:
-            if use_profiling:
-                print(actx.tabulate_profiling_data())
-            print(istep, t, discr.norm(fields[0], np.inf))
-            vis.write_vtk_file("fld-wave-eager-%04d.vtu" % istep,
-                    [
-                        ("u", fields[0]),
-                        ("v", fields[1:]),
-                        ])
+    # initialize Leap integrator if using one
+    import importlib
+    leap_spec = importlib.util.find_spec("leap")
+    not_found = leap_spec is None
+    if not_found:
+        raise ValueError("Leap uninstalled")
+    if use_leap:
+        from leap.rk import RK4MethodBuilder
+        from utils import leap_setup
+        interp = leap_setup(RK4MethodBuilder, "w", rhs, t, dt, fields)
```

Comment:
`interp` is a strange variable name here, particularly for folks who don't know what's going on.

Reply:
Modified (to `stepper_cls`) by 2caff69.